lib/template-loader.php

```diff
@@ -89,6 +89,16 @@ function get_template_hierarchy( $template_type ) {
 function gutenberg_override_query_template( $template, $type, array $templates = array() ) {
 	global $_wp_current_template_content;
 
+	if ( isset( $_GET['_wp-find-template'] ) ) {
+		if ( ! empty( $_GET['_wp-find-template-type'] ) ) {
+			$type = $_GET['_wp-find-template-type'];
+		}
+
+		if ( ! empty( $_GET['_wp-find-template-templates'] ) ) {
+			$templates = $_GET['_wp-find-template-templates'];
+		}
+	}
+
```

Comment:
What would be the purpose of this?

Reply:
We couldn't retrieve templates by specific template types. With these two arguments we can make retrieve a specific template type by specifying the template type and the hierarchy. Hierarchy ($templates) will be the same as $templateType in this case to make sure it returns this specific template, rather than a descendant.